plenoptic/synthesize/eigendistortion.py

```diff
@@ -1,13 +1,14 @@
 import torch
-from ..tools.signal import rescale
+from torch import Tensor
 from .autodiff import jacobian, vector_jacobian_product, jacobian_vector_product
 import numpy as np
 import pyrtools as pt
 import warnings
 from .synthesis import Synthesis
 from tqdm import tqdm
 from matplotlib import pyplot as plt
-import plenoptic as po
+from typing import Tuple
+
```

Comment:
Neither of these imports are used, so they should be removed.

Reply:
since the top got cut off, the "neither" refers to `from typing import Tuple` and `from torch import Tensor`

Reply:
done